iaso/api/data_sources.py

```diff
@@ -114,7 +125,7 @@ def update(self, data_source, validated_data):
             data_source.name = name
 
         read_only = validated_data.get("read_only")
-        if read_only:
+        if read_only is not None:
```

Comment:
Here's the fix: `read_only` is a boolean, so it still needs to be updated when it's `False`